resources/lib/syncEpisodes.py

```diff
@@ -578,10 +575,10 @@ def __compareShows(self, shows_col1, shows_col2, rating=False, restrict=False):
 
                 if show_col2:
                     show = {'title': show_col1['title'], 'ids': {}, 'year': show_col1['year']}
-                    if 'tvdb' in show_col1['ids']:
-                        show['ids'] = {'tvdb': show_col1['ids']['tvdb']}
-                    if 'imdb' in show_col2 and show_col2['imdb']:
-                        show['ids']['imdb'] = show_col2['imdb']
+                    if show_col1['ids']:
+                        show['ids'].update(show_col1['ids'])
+                    if show_col2['ids']:
+                        show['ids'].update(show_col2['ids'])
```

Comment:
It just hasn't been updated to match by multiple ids yet. Also there is no kodi scraper that supports this feature right now. There is a pr pending, but it's on hold due to nobody beeing able to check it.
So the only audience for this change would be people who have nfo files, right now.

I would have to review the code, I'm not sure if it's fixed, that one of those sides always is trakt. Maybe there was even a usecase to `functioncall(trakt,local)` vs `functioncall(local,trakt)`